### PR TITLE
fix news bar alignment

### PR DESCRIPTION
**Pull request by gtalha07**
### Description:
Fixed the like icon alignment.

### CODE REVIEW

**Review by sachintha-s — CHANGES_REQUESTED** [review]
Please tap on the like button and check it out. When it animates it goes to the edge of the device. I think the alignment should be improved when it is animated.

**Comment by gtalha07** [comment]
> Please tap on the like button and check it out. When it animates it goes to the edge of the device. I think the alignment should be improved when it is animated.

Will fix it.

File: app/lib/widgets/NewsItem.dart [hunk 1]
```diff
@@ -37,37 +39,41 @@ class NewsItem extends StatelessWidget {
...
-                      _buildTitle(bgColor, fgColor),
-                      const SizedBox(height: 10),
-                      _buildSubtitle(bgColor, fgColor),
-                      const SizedBox(height: 10),
-                    ],
+        LayoutBuilder(
+          builder: (context, constraints) {
+            return SizedBox(
+              height: constraints.maxWidth >= 600
+                  ? isDesktop
```
Comment (bitfriend) [inline app/lib/widgets/NewsItem.dart:46]: Yeah, desktop layout should be different somewhat from mobile desktop.

**Review by bitfriend — APPROVED** [review]
Looks good to me

**Pull request merged**